Formula/opencascade.rb

```diff
@@ -0,0 +1,91 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://dev.opencascade.org/"
+  url "https://github.com/FreeCAD/homebrew-freecad/releases/download/0/opencascade-7.2.0.tgz"
+  sha256 "f9a81ca2267635773af9fff80625735dd4f65389af986fe93a94263081b422e9"
+
+  bottle do
+    cellar :any
+    sha256 "e817f05ac747235d8453a2d15fb076f90f87b30a872ff6944cc5487f05058cbe" => :sierra
+    sha256 "9be44acf5036801dda4981b48693312f7cba3daffa73e80c50ea207a9da7407e" => :el_capitan
+    sha256 "790964a5e17e0021228f03b32bdabdc1657aa89aa56f668d8b998ec5f1ce42bc" => :yosemite
+    sha256 "5ef4d49225c00242f5ab1cc04804f3d4b931c2fc82713c4a9457e57c50f94c33" => :x86_64_linux
+  end
+
+  option "without-opencl", "Build without OpenCL support"
+  option "with-extras", "Install documentation (~17 MB), source files (~113 MB), samples and templates"
+  option "with-test", "Install tests (~55MB)"
+  deprecated_option "with-tests" => "with-test"
+
+  depends_on "cmake" => :build
+  depends_on "freetype"
+  depends_on "freeimage" => :recommended
+  depends_on "tbb"       => :recommended
+  depends_on "gl2ps"     => :recommended
+  depends_on "doxygen"   if build.with? "extras"
+  depends_on :macos      => :snow_leopard
+
+  conflicts_with "oce", :because => "OCE is a fork for patches/improvements/experiments over OpenCascade"
+
+  def install
+    cmake_args = std_cmake_args
+    cmake_args << "-DCMAKE_PREFIX_PATH:PATH=#{HOMEBREW_PREFIX}"
```

Comment:
Wow, that's a lot of non-standard cmake variables. You sure we need them all?

Reply:
Per instructions here - https://www.opencascade.com/doc/occt-7.2.0/overview/html/occt_dev_guides__building_cmake.html#build_cmake_build